modules/game.feature.menu.map-model.d.ts

```diff
@@ -45,8 +45,8 @@ declare global {
 
       namespace MapEntity {
         interface Settings {
-          mapId?: number,
-          name?: string,
+          mapId: number,
+          name: string,
```

Comment:
`name` is not a required property for entities.

Reply:
Oops, I accidentally selected an old commit for commenting on. I noticed this is still the case in the most recent version, however.